File: application/src/main/java/ru/artemev/application/dto/EmploymentDTO.java
package ru.artemev.application.dto;

import io.swagger.v3.oas.annotations.media.Schema;
import lombok.Data;
import ru.artemev.application.model.enums.EmploymentStatus;
import ru.artemev.application.model.enums.Position;

import java.math.BigDecimal;

@Data
@Schema(description = "Трудоустройство")
public class EmploymentDTO {
  @Schema(description = "Оклад", example = "15000")
  private final BigDecimal salary;

  @Schema(description = "Должность", example = "MID_MANAGER")
  private final Position position;

  @Schema(description = "Общий трудовой стаж", example = "12")
  private final Integer workExperienceTotal;

  @Schema(description = "Текущий трудовой стаж", example = "3")
  private final Integer workExperienceCurrent;

  @Schema(description = "Рабочий статус", example = "SELF_EMPLOYED")
  private EmploymentStatus employmentStatus;

  @Schema(description = "ИНН", example = "123456")
  private String employerINN;
}

File: deal/src/main/kotlin/ru/kit/deal/dto/EmploymentDTO.kt
package ru.kit.deal.dto

import io.swagger.v3.oas.annotations.media.Schema
import java.math.BigDecimal

@Schema(description = "Трудоустройство")
data class EmploymentDTO(
    @Schema(description = "Рабочий статус", example = "SELF_EMPLOYED")
    val employmentStatus: EmploymentStatus? = null,

    @Schema(description = "ИНН", example = "123456")
    val employerINN: String? = null,

    @Schema(description = "Оклад", example = "15000")
    val salary: BigDecimal? = null,

    @Schema(description = "Должность", example = "MANAGER")
    val position: Position? = null,

    @Schema(description = "Общий трудовой стаж", example = "12")
    val workExperienceTotal: Int? = null,

    @Schema(description = "Текущий трудовой стаж", example = "3")
    val workExperienceCurrent: Int? = null,
)